### PR TITLE
ansible for review script

### PR DESCRIPTION
**Pull request by navya091999**
ansible for review script

### CODE REVIEW

File: ansible.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  become: yes
+
+- name: install java package foe centos
+  yum:
+          - java
```
Comment (lokeshdevops2) [inline ansible.txt]: can you add java version 8

File: ansible.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- name: install maven package for centos
+  yum:
+         - maven
```
Comment (lokeshdevops2) [inline ansible.txt]: provide maven version 3.3.9

**Review by lokeshdevops2 — CHANGES_REQUESTED** [review]
can you work on review comments

File: ansible.txt (hunk 2 above)
Comment (navya091999) [reply ansible.txt]: added maven version

File: ansible.txt (hunk 1 above)
Comment (navya091999) [reply ansible.txt]: added java version

**Review by lokeshdevops2 — APPROVED** [review]
code looks good

**Pull request merged**